src/components/BottomModalLayout/RestaurantDetailModal/RestaurantDetailModal.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./RestaurantDetailModal.module.css";
+
+function RestaurantDetailModal({ restaurant, onClose }) {
+  return (
+    <>
+      <h2 className={`${styles.title} text-title`}>{restaurant.name}</h2>
+      <div className={styles.info}>
+        <p className={`${styles.description} text-body`}>
+          {restaurant.description}
+        </p>
+      </div>
+      <div className={styles.buttonContainer}>
+        <button
+          className={`${styles.button} ${styles.buttonPrimary} text-caption`}
+          onClick={onClose}
+        >
+          닫기
+        </button>
```

Comment:
```suggestion
        <button
          type="button"
          className={`${styles.button} ${styles.buttonPrimary} text-caption`}
          onClick={onClose}
        >
          닫기
        </button>
```
버튼의 타입을 명시해줘도 좋을 것 같습니다~🙃

Reply:
? 닫기버튼 submit으로 작동하고 있었겠네... 예리한 피드백 감사합니당ㅎㅎ

Reply:
써밋 버튼 좋긴해요ㅋ 써밋이 기본 값인 이유가 있어요🙃

Reply:
근데 써밋 닉네임 전망대 말고도 submit도 노린거에요?
(진짜 궁금)

Reply:
ㅋㅋㅋㅋ이렇게 나오시는거져?

근데 카멜 닉네임 카라멜 마끼야또 말고도 camelCase도 노린거에요?
(진짜 궁금)

Reply:
? 근데 진짜 이중의미 맞는데...역시나 제 자기소개 안들으셨군요.🫤

Reply:
![ㅇ-ㅇ](https://github.com/user-attachments/assets/4eeae379-8360-4c60-a516-4b0e117c5636)

Reply:
우와 써밋이랑 똑같이 생겼다

Reply:
장`카`설